tool/gcp/tpcc.sh

```diff
@@ -1,4 +1,52 @@
-set -eu
+set -euo allexport
 
-tool/gcp/tpcc-load.sh
-tool/gcp/tpcc-execute.sh
\ No newline at end of file
+usage() {
+    echo "Usage: $0 [-d <neo4j|mongodb|postgres|typedb2|typedb3>] [-s SCALE_FACTOR] [-w WAREHOUSES] [-c CLIENTS] [-t DURATION] [-k]" 1>&2
+    echo "Default: $0 -d typedb3 -s 1 -w 1 -c 1 -t 600" 1>&2
+    echo "-d DATABASE       target database"
+    echo "-s SCALE_FACTOR   benchmark scale factor"
+    echo "-w WAREHOUSES     number of warehouses"
+    echo "-c CLIENTS        number of clients"
+    echo "-t DURATION       how long to run the execution portion of the benchmark (seconds)"
+    echo "-k                keep the server instance (default: deletes)"
+    exit 1
+}
+
+KEEP_SERVER=
+while getopts ":d:w:c:s:t:k" opt; do
```

Comment:
`getopts` can only handle single letter flags. There's also `getopt` (note no `s`) which is a lot more powerful.
However, since `getopts` is a shell builtin and `getopt` is a side binary which behaves differently on different platforms, I'm sticking with this simlpe one.